plugins/extensions/plugins/ZeroEQPlugin.cpp

```diff
@@ -20,6 +20,8 @@
 
 #include "ZeroEQPlugin.h"
 
+#include <ZeroEQ/zeroeq/http/helpers.h>
```

Comment:
won't work if ZeroEQ package is in an installed location.
#include <zeroeq/http/helpers.h>

Reply:
Done